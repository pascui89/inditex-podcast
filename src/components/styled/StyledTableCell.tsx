import {
    TableCell,
    styled,
    tableCellClasses,
  } from '@mui/material';

export const StyledTableCell = styled(TableCell)(({ theme }) => ({
    [`&.${tableCellClasses.head}`]: {
      backgroundColor: theme.palette.common.black,
      color: theme.palette.common.white,
      '&:hover': {
        textDecoration: 'underline',
      }
    },
    [`&.${tableCellClasses.body}`]: {
      fontSize: 14,
      '&:hover': {
        textDecoration: 'underline',
      }  
    },
  }));