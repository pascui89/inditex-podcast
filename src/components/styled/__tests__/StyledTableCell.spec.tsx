import React from 'react';
import { render } from '@testing-library/react';
import { StyledTableCell } from '../StyledTableCell';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledTableCell', () => {

  it('should render', () => {
    const { container } = render(<StyledTableCell/>);
    expect(container).toMatchSnapshot();
  });
});
