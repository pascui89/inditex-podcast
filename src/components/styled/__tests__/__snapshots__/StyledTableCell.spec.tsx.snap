// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTableCell should render 1`] = `
<div>
  <td
    class="MuiTableCell-root MuiTableCell-sizeMedium css-167nyue-MuiTableCell-root"
  />
</div>
`;
